variables.tf

```
@@ -3,6 +3,12 @@ variable "name" {
   description = "The name of the user"
 }
 
+variable "force_create_policy" {
+  type        = bool
+  default     = null
+  description = "Overrule whether the role policy has to be created."
```

This doesn't read nicely, can we change it to something like:

```suggestion
  description = "Force creation of IAM role policy"
```